/sync:
  gnss_message_sync:
    ros__parameters:
      epsilon_sec: 0
      epsilon_nanosec: 1000
/fused:
  gnss_imu_subscriber:
    ros__parameters:
      imu_topic_name: /imu
/**:
  ros__parameters:
    best_topic_name: /best
    bestgnss_topic_name: /bestgnss
    imu_ekf_fused_topic_name: /fused
    difference_best_bestgnss: /diff_best_bestgnss
    difference_best_fused: /diff_best_fused